source/tools/BitVector.h

```diff
@@ -19,6 +19,8 @@
 #define EMP_BIT_VECTOR_H
 
 #include <iostream>
+#include <nmmintrin.h> // testing smth out
```

Comment:
Will this work on systems without this header (non-intel systems?)?
also, what does the comment mean? If it's an old comment, we can remove it